frontend/components/ManageStakingPage/StakingContractSection/index.tsx

```diff
@@ -179,9 +185,17 @@ export const StakingContractSection = ({
       return <AlertNoSlots />;
     }
 
-    if (!hasEnoughOlasToMigrate) {
+    if (
+      !hasEnoughOlasToMigrate &&
+      safeBalance?.OLAS !== undefined &&
+      totalOlasStakedBalance !== undefined
+    ) {
       return (
-        <AlertInsufficientMigrationFunds totalOlasBalance={totalOlasBalance!} />
+        <AlertInsufficientMigrationFunds
+          masterSafeOlasBalance={safeBalance.OLAS}
```

Comment:
can calculate and send only `requiredOlasDeposit` instead of these tree props, they are only used there for calculation but not displayed separately